tests/unit/test_charm.py

```diff
@@ -294,3 +294,20 @@ def test_jvm_heap_memory_config(
             "Invalid jvm-config, "
             "Java heap memory specification exceeds application memory constraint"
         )
+
+
+def test_pebble_ready(
+    harness: Harness,
+    patch: SpringBootPatch,
```

Comment:
If the final `,` is removed from this line `black` should make this a one liner

Reply:
Fixed